app/src/controllers/appInfo/selectors.js

```diff
@@ -1,4 +1,6 @@
 export const appInfoSelector = state => state.appInfo || {};
 
+export const appInfoBuildVersion = state => appInfoSelector(state).UI.build.version || {};
+
```

Comment:
Do not create additional selector. To get UI version use existing appInfoSelector directly in footer component.

Reply:
Why not? Dedicated selector makes it easier to access data.
I just suggest to rename it to `buildVersionSelector`.